webapp/sources/rudder-parent-pom/pom.xml

```diff
@@ -433,6 +435,16 @@ limitations under the License.
         <artifactId>scala-reflect</artifactId>
         <version>${scala-version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.scala-lang.modules</groupId>
+        <artifactId>scala-parser-combinators_${scala-binary-version}</artifactId>
```

Comment:
why do we need it now ?

Reply:
I'm *really* not sure, and actually even if I had a good explanation for that one, I don't understand why it needs to be declared in `rudder-web` since it is a compile dep in `rudder-core`. But it seems that they changed the scope of that module in `scala-compiler` from compile to provided, and so we don't get it through `liftweb-json` (I think).